pkg/apis/pipeline/v1beta1/param_conversion.go

```diff
@@ -48,26 +48,26 @@ func (p *ParamSpec) convertFrom(ctx context.Context, source v1.ParamSpec) {
 
 func (p Param) convertTo(ctx context.Context, sink *v1.Param) {
 	sink.Name = p.Name
-	newValue := v1.ArrayOrString{}
+	newValue := v1.ParamValue{}
 	p.Value.convertTo(ctx, &newValue)
 	sink.Value = newValue
 }
 
 func (p *Param) convertFrom(ctx context.Context, source v1.Param) {
 	p.Name = source.Name
-	newValue := ArrayOrString{}
+	newValue := ParamValue{}
 	newValue.convertFrom(ctx, source.Value)
 	p.Value = newValue
 }
 
-func (aos ArrayOrString) convertTo(ctx context.Context, sink *v1.ArrayOrString) {
+func (aos ArrayOrString) convertTo(ctx context.Context, sink *v1.ParamValue) {
```

Comment:
It seems that the implemented type `aos` should also be changed.

Reply:
thanks!